cli/index.js

```diff
@@ -13,3 +13,23 @@ program.description('Start Jest Preview server.').action(() => {
 });
 
 program.parse(process.argv);
+
+// Checks for available update and notify user
+const updateNotifier = require('update-notifier');
+const chalk = require('chalk');
+
+const notifier = updateNotifier({
+  pkg: require('../package.json'),
+  updateCheckInterval: 0, // How often to check for updates
+  shouldNotifyInNpmScript: true, // Allows notification to be shown when running as an npm script
+  distTag: 'latest' // Can be use to notify user about pre-relase version
+})
+
+notifier.notify({
+  defer: true, // Try not to annoy user by showing the notification after the process has exited
+  message:[
+      `${chalk.blue('{packageName}')} has an update available: ${chalk.gray('{currentVersion}')} → ${chalk.green('{latestVersion}')}`,
+      `Please run ${chalk.cyan('`{updateCommand}`')} to update.`,
+  ].join('\n')
+});
```

Comment:
Can you reformat your code by `npm run prettier:fix`?
You can leave the changes in other files, just need to update code for `cli/index.js` is sufficient.

Reply:
@huynhducduy 
Actually, I think your current code is more readable than the one prettier suggest, you can ignore line 30, 31 if you line.
Something like this
```js
<!-- prettier-ignore-start -->
      `${chalk.blue('{packageName}')} has an update available: ${chalk.gray('{currentVersion}')} → ${chalk.green('{latestVersion}')}`,
      `Please run ${chalk.cyan('`{updateCommand}`')} to update.`,
<!-- prettier-ignore-end -->
```

It's your call. I am good with either option.

Reply:
@nvh95 unfortunately, `prettier-ignore-start` only works in markdown 😢 , so it will need 2 piece of `// prettier-ignore` for those 2 lines if i wanna keep the code in my way, and it does not seem so good 😢

Reply:
I and my team dropped `prettier` also because of problems like that, although `prettier` is simple, it's too opinionated, and sometimes we don't need to strictly format our code like that. IMO, `eslint`'s stylistic rules are enough to play with.